client/src/components/UI/navbar/Navbar.css

```diff
@@ -0,0 +1,125 @@
+.navbar {
+  padding-bottom: 1px;
+  border-bottom: solid 1px #e8e8e8;
+  background-color: white;
+  box-shadow: 0 3px 6px -2px rgba(0, 0, 0, 0.12),
+    0 6px 16px -11px rgba(0, 0, 0, 0.08);
+  position: sticky;
+  top: 0;
+  margin-bottom: 20px;
+  z-index: 99990;
+}
+
+.nav-header {
+  background-color: white;
+  border-bottom: 0;
+  padding: 0 4px 0 24px;
+}
+
+.logo {
+  width: 200px;
+  float: left;
+}
+
+.logo a {
+  display: inline-block;
+  font-size: 20px;
+  text-transform: capitalize;
+}
+
+.navbar-menu {
+  width: calc(100% - 200px);
+  float: left;
+}
+.navbar-menu .ant-layout-header {
+  padding: 0 20px;
+}
+
+.navbar-menu .ant-menu-horizontal {
+  border-bottom: 0;
+}
+
+.navbar-menu .ant-menu-item {
+  height: 4rem;
+  padding: 5px;
+}
+
+.navbar-menu .ant-menu-submenu {
+  height: 4rem;
+  padding: 5px;
+}
+
+.navbar-menu .ant-menu-submenu-title {
+  padding-top: 6px;
+}
+
+.navbar-menu .leftMenu {
+  float: left;
+  padding-top: 1.5rem;
+  margin-left: 5rem;
+}
+
+.navbar-menu .rightMenu {
+  float: right;
+}
+
+.navbar-menu .rightMenu .ant-menu {
+  height: 4rem;
+}
+
+/* Remove underline when hovering */
```

Comment:
I think we can delete the comment lines if we don't need them

Reply:
Yes, you're definitely right, I'll delete them. I haven't finished working on it, should have put it as a draft, my bad!